python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -46,9 +47,9 @@ def __init__(self):
         # The value as reported by the MQTT server at start up of the CSC. This
         # value is reported in the telemetry if no updated values get reported.
         self.initial_value = None
-        # The most recent average as computed during the last
+        # The most recent median as computed during the last
         # HVAC_STATE_TRACK_PERIOD.
-        self.recent_average = None
+        self.recent_median = None
```

Comment:
I am a bit puzzled why you cache this value here, instead of caching it in the telemetry topic, e.g.  calling `tel_<name>.set(<field>=value)`, and then calling `put` when all are set.

It's not a big deal -- what you have works. It just seems slightly indirect to first compute it and then read it back to set the telemetry topic.

Reply:
Yeah good point. I think I did that with checking the value in the telemetry in a unit test in mind but I never got around to implementing that. I'll keep it in mind for a future change.